boards/common/stm32/include/cfg_rtt_default.h

```diff
@@ -31,8 +31,17 @@ extern "C" {
  * On the STM32Lx platforms, we always utilize the LPTIM1.
  * @{
  */
-#define RTT_FREQUENCY       (1024U)             /* 32768 / 2^n */
-#define RTT_MAX_VALUE       (0x0000ffff)        /* 16-bit timer */
+#ifdef LPTIM1
```

Comment:
Why is this needed ?

Reply:
I think it was needed before, vut I fixed that for the `stm32f1` pr changing `rtt`, probably not needed anymore, let me re-check!

Reply:
lets seee if not needed anymore..